bin/test_topology

```diff
@@ -7,14 +7,28 @@ preamble_setup
 # Could be run as its own test shard, if desired.
 bin/test_basics
 
-export FORCH_CONFIG_DIR=topo/bond
+export FORCH_CONFIG_DIR=topo/dva
+export FAUCET_CONFIG_DIR=$OUT_DIR
 
 if [ -z "$VIRTUAL_ENV" -a -f venv/bin/python3 ]; then
     PYTHON=venv/bin/python3
 else
     PYTHON=python3
 fi
 
+function test_faucetizer {
+    test_case=$1
+    fc_file=faucet.yaml
+    ns_file=ns_$test_case.yaml
```

Comment:
ns_ is a bad prefix because it's very ambiguous and misleading. Typically it means "namespace" -- the contents of the file descirbe stuff about devices, so I would prefix with "devices_" or something more meaningful.

Reply:
changed to state_*

Reply:
Changed to devices